entity-services/src/main/xdmp/entity-services/entity-services-impl.xqy

```diff
@@ -1270,7 +1270,14 @@ declare function esi:extraction-template-generate(
 Extraction Template Generated from Entity Type Document
 graph uri: {esi:model-graph-iri($model)}
         </tde:description>
+        <!-- The following line matches JSON and XML instances, but may be slower to index documents. -->
+        <tde:context>//*:instance[*:info/*:version = "{$model=>map:get("info")=>map:get("version")}"]</tde:context>
+        <!-- Replace the above with the following line to match XML instances only.  This may speed up indexing
         <tde:context>//es:instance[es:info/es:version = "{$model=>map:get("info")=>map:get("version")}"]</tde:context>
```

Comment:
Codegen has <tde:context> but the generated template has <context>. Should this be replaced in the comment appropriately. Hope we are asking customers to replace in the generated template and not the code?